contents/tutorials/react-cookie-banner.md

```diff
@@ -2,42 +2,39 @@
 title: Building a tracking cookies consent banner in React
 sidebar: Docs
 showTitle: true
-featuredVideo: 'https://www.youtube-nocookie.com/embed/KRW5jxZ4H-I'
 featuredTutorial: false
-date: 2022-10-03
+date: 2024-06-10
 author:
   - ian-vanagas
 tags:
   - configuration
   - product os
 ---
 
-- **Level:** Medium 🦔🦔
-- **Estimated reading time:** 12 minutes ☕️☕️
+If you’ve spent any time online, you’ve seen a cookie consent banner. Because of GDPR and other worldwide internet privacy regulations, some sites need to get consent to track users and use cookies. Along with this is the need to provide, visitors with an easy way to opt in or out.
```

Comment:
Making a bit shorter 
```suggestion
If you’ve spent any time online, you’ve seen a cookie consent banner. Because of GDPR and other worldwide internet privacy regulations, most sites need to get consent to track users and use cookies. 
```